src/components/views/right_panel/TimelineCard.tsx

```diff
@@ -0,0 +1,104 @@
+/*
+Copyright 2021 The Matrix.org Foundation C.I.C.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import React from 'react';
+import { MatrixEvent, Room } from 'matrix-js-sdk/src';
+import { Thread } from 'matrix-js-sdk/src/models/thread';
+
+import BaseCard from "./BaseCard";
+import { RightPanelPhases } from "../../../stores/RightPanelStorePhases";
+
+import ResizeNotifier from '../../../utils/ResizeNotifier';
+import MessageComposer from '../rooms/MessageComposer';
+import { RoomPermalinkCreator } from '../../../utils/permalinks/Permalinks';
+import { Layout } from '../../../settings/Layout';
+import TimelinePanel from '../../structures/TimelinePanel';
+import { E2EStatus } from '../../../utils/ShieldUtils';
+import EditorStateTransfer from '../../../utils/EditorStateTransfer';
+import RoomContext from '../../../contexts/RoomContext';
+
+import { _t } from '../../../languageHandler';
+import { replaceableComponent } from '../../../utils/replaceableComponent';
+
+interface IProps {
+    room: Room;
+    onClose: () => void;
+    resizeNotifier: ResizeNotifier;
+    permalinkCreator?: RoomPermalinkCreator;
+    e2eStatus?: E2EStatus;
+    initialEvent?: MatrixEvent;
+    initialEventHighlighted?: boolean;
+}
+interface IState {
+    thread?: Thread;
+    editState?: EditorStateTransfer;
+    replyToEvent?: MatrixEvent;
+}
+
+@replaceableComponent("structures.TimelineCard")
+export default class TimelineCard extends React.Component<IProps, IState> {
+    static contextType = RoomContext;
+
+    constructor(props: IProps) {
+        super(props);
+        this.state = {};
+    }
+
+    private renderTimelineCardHeader = (): JSX.Element => {
+        return <div className="mx_TimelineCard__header">
+            <span>{ _t("Chat") }</span>
+        </div>;
+    };
+
+    public render(): JSX.Element {
+        return (
+            <BaseCard
+                className="mx_ThreadPanel mx_TimelineCard"
+                onClose={this.props.onClose}
+                withoutScrollContainer={true}
+                header={this.renderTimelineCardHeader()}>
+                <TimelinePanel
+                    showReadReceipts={true}
```

Comment:
Hmm, I am puzzled by this one being `true`... Weren't we just discussing it's unclear how to display them, so we would leave it off for now?

Reply:
The main concern was that I though it was not easily possible to add this. After the discussion and fixing the error with the optional callback I tried it out and it seemed to behave really well. Thats why I decided to keep it for review to see what you and Gaelle thinks. 
Other than the main timeline the read receipts don't get a dedicated space but just overlap with the message so it does not impact the width of the text and looks really usable:
main timline                                         |                                     right card timeline
![image](https://user-images.githubusercontent.com/16718859/142202759-385edbfe-f4f1-4119-abb3-e970c5d4dfad.png)

Reply:
Should I disable them?
Or can we try them since they of course have their purpose and are not that broken even with smaller timeline widths.
Maybe it is even a good reference for the design team to see what can be improoved?
<img src="https://user-images.githubusercontent.com/16718859/142261694-4d3159aa-cdee-4621-994e-79ce346b2153.png" width="200px"> </img>
^ I finally manged to load enough read receipts in my test account to have a better example.

Reply:
There are cases where it can overlap with the text though. Which maybe is a good reason to disable it for now

Reply:
Yes, I think that's enough evidence that design tweaks are needed for them to work well in a small space, and thus we should disable them for now.